app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,27 @@
+%header#site-header
+  .lg-container.container
+    = link_to root_path, class: "exercism-link" do
+      = icon "exercism-with-logo-black", "Exercism"
+
+    %ul.primary-links
```

Comment:
Is there a reason not to use a `<nav>` tag here?

Reply:
Nope. Added it 🙂